<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AffinityPicker</class>
 <widget class="QWidget" name="AffinityPicker">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelInstructions">
     <property name="text">
      <string>When you nominate a home server for a virtual machine, the virtual machine will always be started up on that server if it is available. If this is not possible, then an alternate server within the same pool will be selected automatically.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wlbWarningWidget" native="true">
     <property name="visible">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="wlbWarningLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="wlbWarningIcon">
        <property name="pixmap">
         <pixmap>:/icons/alert-warning.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="wlbWarningLabel">
        <property name="text">
         <string>WLB is currently enabled. Your home server setting will only be used for VM placement if WLB is disabled.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="dynamicRadioButton">
     <property name="text">
      <string>&amp;Don't assign this VM a home server. The VM will be started on any server with the necessary resources. (Shared storage required).</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="staticRadioButton">
     <property name="text">
      <string>Place the VM on this &amp;server:</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="serversTable">
     <property name="columnCount">
      <number>3</number>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Icon</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Server</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Details</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
